app/controllers/single_use_links_viewer_controller.rb

```diff
@@ -1,69 +1,3 @@
-require 'sufia/single_use_error'
-
 class SingleUseLinksViewerController < ApplicationController
-  include Sufia::DownloadsControllerBehavior
-
-  skip_before_filter :load_file, except: :download
-
-  class Ability
-    include CanCan::Ability
-
-    attr_reader :single_use_link
-
-    def initialize(user, single_use_link)
-      @user = user || User.new
-
-      @single_use_link = single_use_link
-
-      can :read, ActiveFedora::Base do |obj|
-        single_use_link.valid? && single_use_link.itemId == obj.id && single_use_link.destroy!
-      end if single_use_link
-    end
-  end
```

Comment:
Original behavior is here. With my limited understanding of CanCan, it seems to be as long as the single_use_link exists and valid, any current user or new user would be able to read. Not too sure how to validate an instance of single_use_link from ability though. Still reading on Cancan.